sys/shell/commands/sc_netif.c

```diff
@@ -73,30 +73,43 @@ static bool _is_iface(kernel_pid_t dev)
     return false;
 }
 
-#ifdef MODULE_NETSTATS_L2
-static int _netif_stats(kernel_pid_t dev, bool reset)
+#if defined(MODULE_NETSTATS_L2) || defined(MODULE_NETSTATS_IPV6)
```

Comment:
Checking for `NETSTATS` (for the current pseudomodule) should also do the trick. Yes, you're right.